docs/main/troubleshooting/troubleshooting-replicated-engine.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: troubleshooting-re
+id: troubleshootingre
```

Comment:
Why this change?

Reply:
The ID "**troubleshooting**" already exists for _Troubleshooting page of Local Engine_. hence, we've created a new ID "**troubleshootingre**" for the _Troubleshooting page of Replicated Engine_.